src/sentry/static/sentry/app/components/events/eventEntries.jsx

```diff
@@ -156,7 +156,9 @@ class EventEntries extends React.Component {
         {!utils.objectIsEmpty(event.errors) && (
           <EventErrors group={group} event={event} />
         )}{' '}
-        {!isShare && <EventCause event={event} orgId={orgId} projectId={project.slug} />}
+        {!isShare && !!group.firstRelease && (
```

Comment:
Is the assumption that you cannot have suspect commits without a release correct?

Reply:
Yes. The commit detection logic will bail early if the provided event does not have a release.

https://github.com/getsentry/sentry/blob/c56295259f00aff5a2254320a09eac622986aa56/src/sentry/utils/committers.py#L166-L175